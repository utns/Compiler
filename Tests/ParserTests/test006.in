var
    a: integer;
begin    
    a := 1;
.
