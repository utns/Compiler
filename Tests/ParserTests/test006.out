Syntax error. Expected 'end' but '.' found: line = 5 pos = 1
